plugins/node/instrumentation-runtime-node/README.md

```diff
@@ -1,4 +1,4 @@
-# OpenTelemetry Node.js Performance Hooks Instrumentation
+# OpenTelemetry Node.js Runtime Instrumentation
```

Comment:
Would we want the title to make clear this is about runtime *metrics*?

Reply:
Good idea, added in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1970/commits/dabe063d1cfe479ba1d8b219e34041c972493175

I also added a note that it may produce additional signals future (though I can really on think of metrics really making sense right now :thinking:)